modules/mod_base/controllers/controller_postback.erl

```diff
@@ -28,6 +28,7 @@
 
 -include_lib("zotonic.hrl").
 
+-define(SIDEJOB_TIMEOUT, 1000).
```

Comment:
Good question.  The sidejob places its result in the page_session queue, which then signals the comet process. In the mean time we also placed the message ack in the queue.  So if we wait longer than the comet-timeout, then the comet will transport the ack.  

So either we wait shortly and let the comet transport the result, or we wait longer and let the comet transport the ack.